cypress/support/pages/mediashelf/pageMediaShelf.js

```diff
@@ -86,29 +85,41 @@ class MediaShelf {
 
 	isAvailableMediaLineCollapsed() {
 		cy.get(MediaShelf.#availableLine).within(() => {
-			cy.get(".v-expansion-panel-text.no-inner-padding")
-				.should("have.css",	"display", "none");
+			cy.get(".v-expansion-panel-text.no-inner-padding").should(
```

Comment:
Selector should be assigned to the static var here.

Reply:
done